engine/src/main/java/org/terasology/rendering/dag/nodes/SimpleBlendMaterialsNode.java

```diff
@@ -22,28 +22,27 @@
 import org.terasology.registry.In;
 import org.terasology.rendering.dag.AbstractNode;
 import org.terasology.rendering.dag.stateChanges.BindFBO;
-import org.terasology.rendering.opengl.FrameBuffersManager;
+import org.terasology.rendering.opengl.DefaultDynamicFBOs;
 import static org.lwjgl.opengl.GL11.GL_BLEND;
 import static org.lwjgl.opengl.GL11.GL_ONE_MINUS_SRC_ALPHA;
 import static org.lwjgl.opengl.GL11.GL_SRC_ALPHA;
+import org.terasology.rendering.opengl.fbms.DynamicFBOsManager;
 
 /**
  * TODO: Add diagram of this node
  */
 public class SimpleBlendMaterialsNode extends AbstractNode {
 
-    private static final String SCENE_OPAQUE_FBO = "sceneOpaque";
-
     @In
     private ComponentSystemManager componentSystemManager;
 
     @In
-    private FrameBuffersManager frameBuffersManager;
+    private DynamicFBOsManager dynamicFBOsManager;
 
 
     @Override
     public void initialise() {
-        addDesiredStateChange(new BindFBO(SCENE_OPAQUE_FBO));
+        addDesiredStateChange(new BindFBO(DefaultDynamicFBOs.READ_ONLY_GBUFFER.getName(), dynamicFBOsManager)); // TODO: might be removed, verify it
```

Comment:
Can BindFBO use default FBOs directly, to write 

new BindFBO(READ_ONLY_GBUFFER, dynamicFBOsManager)

?